rel-eng/tito.props

```diff
@@ -462,6 +441,28 @@ whitelist = ansiblerole-foreman_scap_client
   rubygem-smart_proxy_remote_execution_ssh
   rubygem-smart_proxy_salt
   rubygem-smart_proxy_spacewalk
+  rubygem-systemu
+  rubygem-timeliness
+  rubygem-ttfunk
+  rubygem-vault
+  rubygem-wicked
+  rubygem-zscheduler
+
+[foreman-plugins-nightly-nonscl-rhel7]
+disttag = .el7
+whitelist = ansiblerole-foreman_scap_client
+  ansiblerole-insights-client
+  foreman-discovery-image-service
+  puppet-foreman_scap_client
+  rubygem-faraday
+  rubygem-fast_gettext
+  rubygem-mysql2
```

Comment:
this is only required by the powerdns smart proxy plugin, and thus does not need a nonscl build now?